docs/intro/cdn.md

```diff
@@ -1,156 +1,122 @@
-npm install @ionic/vue @ionic/vue-router
-```
+<DocsCards>
 
-After that, you will need to install the `IonicVue` plugin in your Vue app.
+<DocsCard
+  header="New JavaScript Project"
+  href="../javascript/quickstart"
+  iconset="/icons/logo-javascript-icon.png,/icons/guide-quickstart-icon.png"
+>
+  <p>Create a new Ionic JavaScript app using Vite.</p>
+</DocsCard>
 
-**main.js**
+</DocsCards>
 
-```javascript
-import { IonicVue } from '@ionic/vue';
-
-import App from './App.vue';
-import router from './router';
-
-const app = createApp(App).use(IonicVue).use(router);
-
-router.isReady().then(() => {
-  app.mount('#app');
-});
-```
-
-Be sure to mount your app once `router.isReady()` has resolved.
-
-### Routing
-
-When setting up routing in your Vue app, you will need to import your dependencies from `@ionic/vue-router` instead of `vue-router`.
-
-**router/index.js**
-
-```javascript
-import { createRouter, createWebHistory } from '@ionic/vue-router';
-
-const routes = [
-  // routes go here
-];
-
-const router = createRouter({
-  history: createWebHistory(process.env.BASE_URL),
-  routes,
-});
-
-export default router;
-```
-
-### CSS
-
-To include the necessary CSS in a Vue project, add the following to your `main.js` file.
+## Ionic Framework CDN
 
-```javascript
-/* Core CSS required for Ionic components to work properly */
-import '@ionic/vue/css/core.css';
+Ionic Framework can be included from a CDN for quick testing in a [StackBlitz](https://stackblitz.com/), [Plunker](https://plnkr.co/), [Codepen](https://codepen.io), or any other online code editor!
 
-/* Basic CSS for apps built with Ionic */
-import '@ionic/vue/css/normalize.css';
-import '@ionic/vue/css/structure.css';
-import '@ionic/vue/css/typography.css';
+It's recommended to use [jsdelivr](https://www.jsdelivr.com/) to access the Framework from a CDN. To get the latest version, add the following inside the `<head>` element in an HTML file, or where external assets are included in the online code editor:
 
-/* Optional CSS utils that can be commented out */
-import '@ionic/vue/css/padding.css';
-import '@ionic/vue/css/float-elements.css';
-import '@ionic/vue/css/text-alignment.css';
-import '@ionic/vue/css/text-transformation.css';
-import '@ionic/vue/css/flex-utils.css';
-import '@ionic/vue/css/display.css';
+```html
+<script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js"></script>
+<script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.js"></script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css" />
 ```
 
-From here, you can learn about how to develop with Ionic Framework in our [Ionic Vue Quickstart Guide](https://ionicframework.com/docs/vue/quickstart).
+With this it's possible to use all of the Ionic Framework core components without having to install a framework. The CSS bundle will include all of the Ionic [Global Stylesheets](../layout/global-stylesheets).
```

Comment:
I noticed that in Angular Add to Existing uses `/docs` for the internal links but in this example, it's using `...`. Why is that? Maybe we should stick to one?


```suggestion
With this it's possible to use all of the Ionic Framework core components without having to install a framework. The CSS bundle will include all of the Ionic [Global Stylesheets](../layout/global-stylesheets.md).
```